os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_central/ble_tizenrt_client.c

```diff
@@ -74,6 +77,167 @@ trble_result_e rtw_ble_client_init(trble_client_init_config* init_parm)
     return TRBLE_SUCCESS; 
 }
 
+
+trble_result_e rtw_ble_client_scan_whiltelist_add(trble_addr *addr)
+{
+    if (client_init_parm == NULL || client_init_parm->trble_device_scanned_cb == NULL || addr == NULL)
+    {
+        return TRBLE_FAIL;
+    }
+
+    if (ble_tizenrt_modi_whitelist_sem == NULL)
+    {
+        if(false == os_mutex_create(&ble_tizenrt_modi_whitelist_sem))
+        {
+            printf("\r\n[%s] create sem fail!", __FUNCTION__);
```

Comment:
There are `printf` and `debug_printf`.
I think it is better to use `debug_printf`.
How do you think?

Reply:
There is other printf in the RTK BT files.
debug_printf it is also a printf as well, so i'm preparing a PR to replace all debug message with TizenRT format.